Epomaker tide75

Epomaker opened the pull request:
<!--- Provide a general summary of your changes in the title above. -->

<!--- This template is entirely optional and can be removed, but is here to help both you and us. -->
<!--- Anything on lines wrapped in comments like these will not show up in the final text. -->

## Description

<!--- Describe your changes in detail here. -->

## Types of Changes

<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply. -->
- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

waffle87 commented on keyboards/epomaker/tide75/info.json:
```diff
@@ -0,0 +1,267 @@
+    "keyboard_name": "EPOMAKER TIDE 75",
+    "maintainer": "sdk66",
+    "bootloader": "wb32-dfu",
+    "bootmagic": {
+        "matrix": [0, 0]
+    },
```
```suggestion
```

Epomaker commented:
@waffle87 
I have completed the work on this Pull Request and am ready to merge it into the main branch. However, I have found that the merge operation is currently restricted, and only authorized users can perform it.

Could you please help me merge this Pull Request? If you have any questions or need further information, please feel free to let me know. Thank you!

adophoxia commented:
@Epomaker 
> However, I have found that the merge operation is currently restricted, and only authorized users can perform it.

This will only be for people like the collaborators like waffle87, or more so directors like tzarc. So only *they* have the power for merging PRs like this.

> Could you please help me merge this Pull Request?

The easiest way you can make this PR go smoothly is if you follow the requested changes from any reviews made, and to not leave them unnoticed for months as some PRs in the past have done so.

Epomaker commented:
> @Epomaker
> 
> > However, I have found that the merge operation is currently restricted, and only authorized users can perform it.
> 
> This will only be for people like the collaborators like waffle87, or more so directors like tzarc. So only _they_ have the power for merging PRs like this.
> 
> > Could you please help me merge this Pull Request?
> 
> The easiest way you can make this PR go smoothly is if you follow the requested changes from any reviews made, and to not leave them unnoticed for months as some PRs in the past have done so.

I sincerely appreciate your detailed feedback and guidance. I'm committed to meticulously addressing the reviewers' suggestions and actively monitoring the progress of this PR. With our collective effort, I'm confident we can swiftly merge this PR.

Epomaker commented:
@waffle87 
Hi there! I've just uploaded my code. I'd be grateful if you could take a look and provide some feedback. If you see any areas for improvement, please don't hesitate to let me know. I'm eager to learn and make the necessary changes as soon as possible. Thanks in advance!

drashna commented on keyboards/epomaker/tide75/info.json:
```diff
@@ -0,0 +1,264 @@
+        "force_nkro": true,
+        "pid": "0xE471",
+        "suspend_wakeup_delay": 1000,
+        "vid": "0x342D"
```
This looks to be Westberry's vendor ID.  Do you have permission from them to use this VID+PID?

Epomaker replied:
Yes, we have permission to use it. We applied to Westberry.

tzarc commented:
Converted to draft and marked with `needs-wireless-code` due to Tide65 being merged without wireless bindings.
All Epomaker wireless boards are now on hold until Tide65 is rectified.

tzarc commented:
This PR is on hold until #24085 is rectified.

tzarc commented:
As of August 26, 2024, `qmk/qmk_firmware` is no longer accepting VIA-enabled keymaps as these have now transitioned to a repository under the VIA team's control.

As you've submitted a PR containing `via` or VIA-enabled keymap(s), this is your notice that they should be removed from this PR. You should now submit a secondary PR to the [VIA QMK Userspace](https://github.com/the-via/qmk_userspace_via) repository with your associated `via` or VIA-enabled keymaps instead.
